lib/vdsm/virt/qemuguestagent.py

```diff
@@ -478,7 +488,11 @@ def _poller(self):
                         not after_hotplug:
                     continue
                 # Commands that have special handling go here
-                if command == VDSM_GUEST_INFO_DRIVERS:
+                if command == VDSM_GUEST_INFO_CPUS:
+                    self.update_guest_info(
+                        vm_id, self._qga_call_get_vcpus(vm_obj))
+                    self.set_last_check(vm_id, command, now)
```

Comment:
This starts calling for refactoring, e.g.:

``` python
special_commands = {
    VDSM_GUEST_INFO_CPUS: self._qga_call_get_vcpus,
    VDSM_GUEST_INFO_DRIVERS: self._qga_call_get_devices,
    VDSM_GUEST_INFO_NETWORK: self._qga_call_network_interfaces,
}
special_handler = special_commands.get(command)
if special_handler is not None:
    self.update_guest_info(vm_id, special_handler(vm_obj))
    self.set_last_check(vm_id, command, now)
```

Reply:
Yes, I'll open a separate PR for refactoring.

Reply:
OK, thanks.